timetracker/timetracker/urls.py

```diff
@@ -18,16 +18,14 @@
 from django.http import HttpResponse
 
 
-def hello_world(request):
-    name = request.GET.get('name')
-    if name:
-        return HttpResponse('Hello %s' % name)
-    else:
-        return HttpResponse('<form><input name="name"></form>')
-
-
+import entries.views
 
 urlpatterns = [
-    url(r'^$', hello_world),
+    url(r'^/$', entries.views.index, name="home"),
+    url(r'^clients/$', entries.views.client_list, name="client_list"),
+    url(r'^clients/(?P<pk>\d+)/$', entries.views.client_detail, name="client_detail"),
+    url(r'^clients/summary/(?P<pk>\d+)/$', entries.views.client_summary, name="client_summary"),
```

Comment:
A lot of people would express this as:
```
r'^clients/(?P<pk>\d+)/summary/
```
Think of it as greatest to narrowest scope : all clients, client with this pk, summary of client with this pk...